esx_service/utils/kvESX.py

```diff
@@ -123,42 +198,57 @@ def volOpenPath(volpath):
 
 # Create the side car for the volume identified by volpath.
 def create(volpath, kv_dict):
-    disk = c_uint32(0)
-    objHandle = c_uint32(0)
+    if using_64bits:
```

Comment:
same comment - please wrap c_uint conversion and use it instead of multiple  'if using_64bits:'